webapp/app/components/Main.test.js

```diff
@@ -1,19 +1,50 @@
 import React from 'react';
 import { shallow } from 'enzyme';
-import { Main } from './Main'; // Undecorated version.
+import { keys } from 'lodash/fp';
+import { Main, mapDispatchToProps, mapStateToProps } from './Main'; // Undecorated version.
+import initialState from '../reducers/initialState';
 
 function setup(props) {
   return shallow(<Main {...props} />);
 }
 
 describe('<Main /> component', () => {
   it('renders itself', () => {
+    // Arrange
     const wrapper = setup({
       actions: {},
       usersActions: {}
     });
 
+    // Assert
     expect(wrapper.find('Switch')).toHaveLength(1);
     expect(wrapper.find('Route')).toHaveLength(3);
   });
+
+  describe('mapToProps functions', () => {
+    it('should return the state of auth module', () => {
+      // Act
+      const props = mapStateToProps(initialState);
+
+      // Assert
+      expect(props.isAuthenticated).toBe(false);
+    });
+
+    it('should return the binded actions ', () => {
+      // Arrange
```

Comment:
Please remove the commented out code, if comments are to be commited, we should make sure they serve a purpose, i.e. to explain a non obvious solution, as a general rule we avoid obvious comments to not pollute the code.